Makefile

```diff
@@ -163,6 +163,11 @@ nocover:
 	@echo Verifying that all packages have test files to count in coverage
 	@scripts/check-test-files.sh $(ALL_PKGS)
 
+.PHONY: goleak
+goleak:
```

Comment:
Add this target as a dependency of lint (see how nocover target is called)

Reply:
do you mean like this
 ```make
 .PHONY: cover
cover: nocover goleak
	$(GOTEST) -tags=memory_storage_integration -timeout 5m -coverprofile cover.out ./...
	go tool cover -html=cover.out -o cover.html`